packages/terra-embedded-content-consumer/docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Terra Embedded Content Consumer
+
+The Embedded Content Consumer is the application component which is embedding content within it.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-embedded-content-consumer`
+  - `yarn add terra-embedded-content-consumer`
+
+## Usage
+
+```jsx
+import React from 'react';
+import EmbeddedContentConsumer from 'terra-embedded-content-consumer';
+
+<EmbeddedContentConsumer
+  src="https://google.com"
+  onLaunch={() => { console.log('launched'); }}
+  onAuthorize={() => { console.log('Authorized'); }}
+  onFullScreen={() => { window.open('https://yahoo.com'); }}
```

Comment:
My preference would be to remove this functionality from this PR if there is consensus that this should be handled via the custom event handlers.